src/nspawn/nspawn.c

```diff
@@ -2186,6 +2239,21 @@ static int copy_devnodes(const char *dest) {
                 _cleanup_free_ char *from = NULL, *to = NULL;
                 struct stat st;
 
+                if (streq(d, "fuse")) {
+                        /* FUSE is only userns-safe in FUSE version 7.27 and later.
+                         * https://github.com/torvalds/linux/commit/da315f6e03988a7127680bbc26e1028991b899b8 */
+                        uint32_t major, minor;
+                        if (get_fuse_version(&major, &minor) < 0) {
+                                /* get_fuse_version() already logged a warning. */
+                                continue;
+                        }
+                        if (major < 7 || (major == 7 && minor < 27)) {
+                                log_warning("Disabling FUSE: FUSE version %" PRIu32 ".%" PRIu32 " is too old to support user namespaces",
+                                            major, minor);
```

Comment:
this should be a debug log message at best.

Reply:
Very well. My rationale for using LOG_WARN was that "something you may otherwise have expected to work isn't going to work" is worth warning about, but I don't have a real strong opinion about that. I almost went with LOG_NOTICE to match `log_notice("Automatically implying --private-network, since mounting /sys/ in an unprivileged user namespaces requires network namespacing.");`.

Reply:
well, but fuse is a relatively exotic feature. I never cared for it. You do. We shouldn't assume that everyone cares, it's probably a minority, and hence this deserves mention in the debug output but not always.

Reply:
Saying that I "care for it" sounds quite a bit warmer about FUSE than my actual feelings toward it... :-)